### ISSUE TITLE
Unexpected behavior of LabeledVideoDataset class

### ISSUE THREAD

**Issue by anisaha1** [facebookresearch/pytorchvideo#100]
Hi, @kalyanvasudev

I have a simple scenario like this.
A csv file with 101 UCF video paths and labels, one video from each class.
I use the pytorchvideo.data.Ucf101 class to create my dataset and then the dataloader.
I use uniform clip sampler and the rest of the arguments are defaults.
The iterable UCF101 dataset is wrapped in LimitDataset class.
My clip duration is (8x8)/30 ~ 2.13.

When I enumerate the Dataloader, I expect it to return 101 samples per epoch and each sample to be one clip per video with clip start 0.0 and clip end 2.13. 
But it returns multiple clips for each video and with no apparent pattern that I could decipher.
I was expecting a deterministic clip sampling. 

Is there something I do not understand here? And can you please tell me how do I achieve a deterministic 0.0 - 2.13 one clip/video sampling?
